### PR TITLE
An attempt at trying to fix or at least debug #24418 by preloading.

### PR DESCRIPTION
**Pull request by N3X15**
Refer to #24418.  Here I make use of the same preloading TG uses for their VOX.  This may not work very well since there are 4,500 oggs to load and stoplag() is called between them.

## Changelog

:cl:
 * bugfix: Try to fix VOX announcements by preloading sounds.  If they load out of order, players receive jumbled announcements.
 * wip: Added debugging information that pops up a notice telling the AI the words it received in the announcement.

### CODE REVIEW

**Comment by N3X15** [comment]
CURSE YOU SECRET REPOOOOOOO

**Comment by Probe1** [comment]
This.

**Comment by Intigracy** [comment]
that's just a collab's problem when they merge it

File: code/modules/client/client procs.dm [hunk 1]
```diff
@@ -381,6 +381,21 @@
 	spawn()
 		if(src in clients) //Did we log out before we reached this part of the function?
 			nanomanager.send_resources(src)
+	// Hahaha your suffering is not yet over, as we still must waterboard you with ten thousand oggs! (4500 at last count)
+	spawn()
+		if(src)
```
Comment (DamianX) [inline code/modules/client/client procs.dm:386]: iirc there's no way for this to fail other than manually setting `src = null`
when the datum gets deleted its scheduled procs just get removed from the queue

**Comment by DamianX** [comment]
If this makes clients too slow to connect, maybe we could try delaying the vox announcements by `some_amount` and only preload the files that are needed during that delay

File: code/modules/client/client procs.dm (hunk 1 above)
Comment (N3X15) [reply code/modules/client/client procs.dm:386]: Actually, during testing, it would fail if you logged during loading.

Comment (DamianX) [reply code/modules/client/client procs.dm:386]: I guess clients work differently

**Comment by N3X15** [comment]
That works without lag over my LAN, but I don't know how it'll perform over the intertubes. We'll see, I guess.

**Comment by DamianX** [comment]
Lags too much, revert when

**Pull request merged**